ewardrobe/ewardrobe_app/queries/orders.py

```diff
@@ -0,0 +1,31 @@
+from ewardrobe_app.models import Basket
+from django.contrib.auth.models import User
+from ewardrobe_app.models import STATUS_CHOICES
+
+
+TRANSITIONS = {
+    0: ["Pay", "Cancel"],
+    1: ["Shiping"],
+    2: ["Close", "Return"],
+}
+
+
+class OrderWorkflow:
+    def __init__(self, user: User):
+        self.__user = user
+
+    def prepare_baskets(self) -> dict:
+        orders = Basket.objects.filter(user=self.__user).order_by("-date_created").all()
+        orders_data = []
+        for idx, order in enumerate(orders):
```

Comment:
idk, but maybe you can avoid calling `all()` in a line before and simply `enumerate` on the query?

Reply:
done